script_runner/tests/test_ctx_wrapper.py

```diff
@@ -142,8 +142,8 @@ def test_logger(self, capture):
             'error_message'
         ]
         self._run(script)
-        # first message is unrelated to the test
-        capture.records.pop(0)
+        # first two messages are unrelated to the test
+        del capture.records[:2]
```

Comment:
is this changed here as well, or is this just a leftover from the previous PR?

Reply:
Changed here as well. This is not related to the previous PR. When both are merged, this will be changed to `[:3]`.